.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    working_directory: ~/eventuate-tram-examples-vanilla-event-publishing
+    steps:
+      - checkout
+      - restore_cache:
```

Comment:
See this version of restore cache, which has multiple keys: https://github.com/microservices-patterns/ftgo-application/blob/9f85c7744bfa2d58a2a9aefbb3b9eb43428df3ec/.circleci/config.yml#L8

This should be used in config.yml

Reply:
ok